modules/processing/redshift/doc/ST_POLYGONIZE.md

```diff
@@ -0,0 +1,22 @@
+### ST_POLYGONIZE
```

Comment:
BTW, this in BQ does the same that `ST_MAKEPOLYGON` right? Apart from accepting arrays do you know if there is any other reason to have it?

I'm also curious why the naming is not something more similar to the other naming, for instance: `ST_MAKEPOLYGON_ARR`.

Reply:
I think it was added because it appears in PostGIS with that name, however we can delete this if @Jesus89 agrees. For now, it's a simple alias. Let's see in QA